web/public/js/directives/cd-action-bar/cd-action-bar-checkbox-item/style.less

```diff
@@ -0,0 +1,22 @@
+.cd-action-bar-checkbox-item {
```

Comment:
A lot of duplicate CSS. Would be nice to use base classes from the standard action bar items. That would put a dependency between components that wouldn't otherwise be related.